phpbench.json

```diff
@@ -6,7 +6,7 @@
             "compare": "benchmark",
             "compare_fields": [ "mode" ],
             "cols": [ "subject" ],
-            "labels": [ "Subject", "VDB", "Aura", "NetUrl", "ZendUri", "League", "Windwalker" ]
+            "labels": [ "Subject", "VDB", "Aura", "Dicrators", "NetUrl", "ZendUri", "League", "Guzzle", "Slime", "Windwalker" ]
```

Comment:
Good catch I hadn't updated this part there are some typos:
- Dicrators -> Zend Diactoros
- Slime -> Slim